### ISSUE TITLE
Load multiple SWFs from a single zip

### ISSUE THREAD

**Issue by urain39** [ruffle-rs/ruffle#11036]
**Describe the feature**

It allows web pages to load games containing multiple files.

**Comment by Lord-McSweeney** [comment]
I don't understand, is this for the web version of Ruffle? Will this feature just add multiple Ruffle containers to the page, each running a SWF?

**Comment by urain39** [comment]
Yes, it is for web version. But not each running a SWF, it has a main entry like main.swf.

**Comment by p0008874** [comment]
> **Describe the feature**
> 
> It allows web pages to load a game containing multiple files, and with a main entry (such as main.swf).

This more likely the zlib can do.
I  play lot M.U.G.E.N and I understand that reference.

**Comment by n0samu** [comment]
Why can't you just upload all of the SWF files to your server and load the main.swf file in Ruffle? That is how the games were designed to be loaded.

**Comment by urain39** [comment]
You're right, but this may be a bit complicated for normal people. 🤔
